src/components/CommentReply/style.scss

```diff
@@ -1,60 +1,119 @@
 .comment-reply {
-    &__header {
-        font-size: 1.2rem;
-
-        hr {
-            background-color: #ccc;
-            margin-top: 0;
-            margin-bottom: 0;
-            margin-left: 10px;
-            margin-right: 10px;
-            border: 0;
-            height: 2px;
+    margin-top: 20px;
+    border-left: 5px solid #5b5b5b;
+    padding-left: 7px;
+
+    &__text {
+        color: #fff;
+        font-size: 1em;
+        font-weight: bold;
+        margin-top: 12px;
+        margin-bottom: 0px;
+
+        &--mode-deleting {
+            color: #333;
         }
     }
 
-    &__date {
-        font-size: 1.2rem;
-        color: #555;
-        margin: 0;
+    &__author-date {
+        color: #fff;
+        font-size: 0.8em;
+        margin-top: 0px;
+        margin-bottom: 12px;
     }
 
-    &__header-info {
-        float: left;
-        padding: 10px;
-
-        h2 {
-            margin: 0;
-            font-weight: bold;
-            color: #555;
-        }
+    &--mode-deleting &__text,
+    &--mode-deleting &__author-date {
+        color: #999;
     }
 
-    &__header-actions {
-        float: right;
-        padding: 10px;
+    &__button {
+        background-color: $background;
+        border: 1px solid #626262;
+        border-radius: 3px;
+        color: #fff;
+        cursor: pointer;
+        text-transform: uppercase;
 
-        > a {
-            display: inline-block;
-            padding-right: 4px;
+        &--primary {
+            border: 1px solid #007d7e;
+            background-color: #007d7e;
+        }
 
-            color: #333;
+        &--red {
+            border: 1px solid #901419;
+            background-color: #901419;
         }
     }
 
-    &__confirm-delete {
+    &__actions,
+    &__confirm-delete,
+    &__progress {
+        height: 30px;
+    }
+
+    &__actions &__button,
+    &__confirm-delete &__button {
+        height: 26px;
+        margin-right: 5px;
+        padding: 6px;
+        padding-bottom: 4px;
+        padding-left: 9px;
+        padding-right: 9px;
+        margin-top: 5px;
         margin-right: 5px;
     }
 
+    &__confirm-delete,
     &__error {
-        color: red;
+        color: #fff;
         font-weight: bold;
+        font-size: 0.8em;
+
+        button {
+            margin-left: 5px;
+            margin-right: 0px;
+        }
     }
 
-    &__text {
-        clear: both;
-        padding-left: 20px;
-        padding-bottom: 10px;
+    &__error {
+        background-color: #b4191f;
```

Comment:
Maybe worth replacing hardcoded colours for maintainability?

Reply:
Done!